src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -11,10 +11,11 @@
 			<h1>DevOps Traning	12 09 2020</h1>
 			<h1>DevOps Traning  12 09 2020</h1>
 			<h1>DevOps Traning Mohanraj Vellingiri</h1>
-			<h1> Devops Training Nallaperumal TN	</h1>
-			<h1> Devops Training Nallaperumal TN	</h1>
-			<h1>2 DevOps Training Mohit Kambam</h1>
-			</h1> DevOps Training 12 09 2020 Mohit Kambam</h1>
+			<h1>Devops Training Nallaperumal TN	</h1>
+			<h1>Devops Training Nallaperumal TN	</h1>
+			<h1>DevOps Training Mohit Kambam</h1>
+			<h1> DevOps Training 12 09 2020 Mohit Kambam</h1>
+			My New Code is here
```

Comment:
Please update the code with Training name

Reply:
Use capital letters

Reply:
Ok